test-integration/src/test/java/nl/nn/adapterframework/pipes/EsbSoapWrapperPipeTest.java

```diff
@@ -69,15 +67,15 @@ public void testWrapFindDocumentsOK() {
 		String operation="FindDocuments";
 		String operationVersion="1";
 		String paradigm="Request";
-		
+
 		String errorCode=null;
 		String errorReason=null;
 		String errorDetailCode=null;
 		String errorDetailText=null;
-		
-		
+
+
 		String destination=messagingLayer+"."+businessDomain+"."+applicationName+"."+serviceName+"."+serviceVersion+"."+operation+"."+operationVersion+"."+paradigm;
-		
+
```

Comment:
Wil je ook in deze classe de overbodige lege regels verwijderen? Wel laten staan als het handig is qua leesbaarheid, voor het onderscheid tussen test-setup en test-assertions bv.